### PR TITLE
Adjust some translations

### LINKED ISSUE #857: Wording: DE/EN coherence
Can you extract the relevant "wording" so that we can take an overall look instead of discussing it here (Dashboard) and there (metadata/upload)?

### PR DESCRIPTION
**Pull request by owi92**
Closes #857

### CODE REVIEW

**Review by LukasKalbertodt — APPROVED** [review]
Changes look good to me! I will wait with merging until we have a response from Olaf.

File: frontend/src/i18n/locales/de.yaml [hunk 1]
```diff
@@ -437,7 +437,7 @@ manage:
         failed: Löschung fehlgeschlagen.
         warning: >
           Wenn Sie diese Seite löschen, werden automatisch alle direkten und
-          indirekten Unterseiten sowie alle Seiteninhalte all dieser Seiten
+          indirekten Unterseiten sowie alle Seiteninhalte dieser Seiten
```
Comment (owi92) [inline frontend/src/i18n/locales/de.yaml:440]: Maybe even 
```suggestion
          indirekten Unterseiten sowie alle Inhalte dieser Seiten
```

Comment (LukasKalbertodt) [reply frontend/src/i18n/locales/de.yaml:440]: Ye I suppose "all" is really not necessary. And +1 for "Inhalte".

**Review by LukasKalbertodt — APPROVED** [review]
Olaf also seems to be happy with these changes. So let's merge. We can of course always add more adjustments in the future.

**Pull request merged**